homeassistant/components/wemo/__init__.py

```diff
@@ -16,9 +15,14 @@
 from homeassistant.helpers import config_validation as cv
 from homeassistant.helpers.dispatcher import async_dispatcher_send
 from homeassistant.helpers.event import async_call_later
+from homeassistant.util.async_ import gather_with_concurrency
 
 from .const import DOMAIN
 
+# Max number of devices to initialize at once. This limit is in place to
+# avoid tying up too many asyncio executors with WeMo device setup.
```

Comment:
executor threads instead of asyncio executors since the executor can be used without asyncio